website/content/docs/connect/config-entries/sameness-group.mdx

```diff
@@ -317,7 +342,7 @@ In the following example, the configuration entry defines a sameness group named
 Kind                     = "sameness-group"  
 Name                     = "products-api"
 Partition                = "store-east"
-DefaultForFailover = true
+IncludeLocal             = true
```

Comment:
Because `store-east` already exists in the member list, this `IncludeLocal = true` will throw a validation error stating that there are duplicate entries in the sameness group. Typically, I would steer users away from the `IncludeLocal = true` and simply have them define the list explicitly to avoid confusion.